x-pack/plugins/uptime/public/components/monitor/synthetics/waterfall/components/middle_truncated_text.tsx

```diff
@@ -80,7 +81,13 @@ export const MiddleTruncatedText = ({ ariaLabel, text, onClick, setButtonRef, ur
       <EuiScreenReaderOnly>
         <span data-test-subj="middleTruncatedTextSROnly">{text}</span>
       </EuiScreenReaderOnly>
-      <EuiToolTip content={text} position="top" data-test-subj="middleTruncatedTextToolTip">
+      <WaterfallTooltipResponsiveMaxWidth
+        as={EuiToolTip}
```

Comment:
TIL. Does this mean that the div in `euiStyled.div` is essentially overwritten and the wrapper now becomes EuiToolTip?

Reply:
Exactly that; very cool helper prop for composability/reusability.